Assets/Scripts/System/GM.cs

```diff
@@ -20,6 +21,7 @@ public class GM : Singleton<GM> {
         options = GetComponent<OptionsManager> ();
 		state = GetComponent<StateManager> ();
         dbug = GetComponent<Dbug>();
+        sync = GetComponent<GameSync>();
```

Comment:
It's not, though. The GameSync component doesn't (or hasn't yet?) been initialized. So when trying to run the sync I get this:
<img width="506" alt="screen shot 2016-10-17 at 9 01 27 pm" src="https://cloud.githubusercontent.com/assets/1580121/19461932/6f69cb44-94ae-11e6-95da-10545e5e3751.png">